content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -182,14 +182,14 @@ kubelet [flags]
 <td colspan="2">--container-log-max-files int32&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 5</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">&lt;Warning: Beta feature&gt; Set the maximum number of container log files that can be present for a container. The number must be &gt;= 2. This flag can only be used with <code>--container-runtime=remote</code>. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">&lt;Warning: Beta feature&gt; Set the maximum number of container log files that can be present for a container. The number must be &gt;= 2. (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
 </tr>
 
 <tr>
 <td colspan="2">--container-log-max-size string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: <code>10Mi</code></td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">&lt;Warning: Beta feature&gt; Set the maximum size (e.g. <code>10Mi</code>) of container log file before it is rotated. This flag can only be used with <code>--container-runtime=remote</code>.  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">&lt;Warning: Beta feature&gt; Set the maximum size (e.g. <code>10Mi</code>) of container log file before it is rotated.  (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)</td>
```

Comment:
@pacoxu are you sure the content is like that in the kubelet cli?
@tengqm perhaps the generator did not run?

Reply:
Well ... I have been unable to generate reference for `kubelet` because it invented a new way to emit help options, those options are only available when you run `kubelet -h`. So, we have been maintaining the kubelet reference for over two years.

Reply:
yes,  I checked my local kubelet 

> [root@daocloud ~]# ./kubelet --version
> Kubernetes v1.27.2
> [root@daocloud ~]# ./kubelet --help | grep  container-log-max-size
>       --container-log-max-size string                            <Warning: Beta feature> Set the maximum size (e.g. 10Mi) of container log file before it is rotated. (default "10Mi") (DEPRECATED: This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.)

https://github.com/kubernetes/kubernetes/pull/114017 is merged in v1.27.

Reply:
My question is about the logging options. Since the related feature gate `CRIContainerLogRotation` has been GA'ed in 1.21 and removed in 1.23, are we still treating this option as Beta?

Reply:
> My question is about the logging options. Since the related feature gate `CRIContainerLogRotation` has been GA'ed in 1.21 and removed in 1.23, are we still treating this option as Beta?

Good Question, I think we should remove the warning here.
But before removing it here, we should do a cleanup in k/k code base first.